lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -815,7 +818,7 @@ SPIRVEntry *LLVMToSPIRVDbgTran::transDbgSubrangeType(const DISubrange *ST) {
                          : getDebugInfoNoneId();
     }
   };
-  for (int Idx = CountIdx; Idx < OperandCount; ++Idx)
+  for (int Idx = 0; Idx < MaxOperandCount; ++Idx)
     TransOperand(Idx);
   return BM->addDebugInfo(SPIRVDebug::TypeSubrange, getVoidTy(), Ops);
```

Comment:
Consider removing any special treatment of StrideIdx in TransOperand and just resizing to MinOperandCount before the return if Ops[StrideIdx] == getDebugInfoNoneId().